docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-implemenation-entity-framework-core.md

```diff
@@ -448,7 +448,7 @@ Although we don’t recommend to return IQueryable from a repository, it’s per
   [*https://docs.microsoft.com/ef/core/modeling/relational/tables*](https://docs.microsoft.com/ef/core/modeling/relational/tables)
 
 - **Use HiLo to generate keys with Entity Framework Core** \
-  [*http://www.talkingdotnet.com/use-hilo-to-generate-keys-with-entity-framework-core/*](http://www.talkingdotnet.com/use-hilo-to-generate-keys-with-entity-framework-core/)
+  [*https://www.talkingdotnet.com/use-hilo-to-generate-keys-with-entity-framework-core/*](https://www.talkingdotnet.com/use-hilo-to-generate-keys-with-entity-framework-core/)
```

Comment:
This has the same problem, with the link text continuing to be rendered as "http":

![image](https://user-images.githubusercontent.com/10886961/50783582-29dcb000-1260-11e9-9f4d-78fa72b30ca7.png)